pyvrp/educate/LocalSearch.py

```diff
@@ -0,0 +1,52 @@
+from typing import List
+
+from pyvrp.Individual import Individual
+from pyvrp.PenaltyManager import PenaltyManager
+from pyvrp.ProblemData import ProblemData
+from pyvrp.XorShift128 import XorShift128
+from ._LocalSearch import LocalSearch as _LocalSearch
+
+Neighbours = List[List[int]]
+
+
+class LocalSearch:
+    def __init__(
+        self,
+        data: ProblemData,
+        penalty_manager: PenaltyManager,
+        rng: XorShift128,
+        neighbours: Neighbours,
+    ) -> None:
+        """
+        Python wrapper for a LocalSearch instance.
+
+        Parameters
+        ----------
+        data
+            Data object describing the problem to be solved.
+        penalty_manager
+            Penalty manager to use.
+        rng
+            Random number generator.
+        neighbours
+            Matrix that defines the local search neighbourhood.
```

Comment:
`neighbours` need not be a matrix, it's just a list of (possibly variable length) lists.